docker-bake.hcl

```diff
@@ -1,29 +1,75 @@
-variable "REPO_NAME" {
+variable "IMAGE_NAME" {
     default = "dunglas/frankenphp"
 }
 
 group "default" {
-    targets = ["bullseye", "alpine"]
+    targets = ["bullseye", /*"buster", "alpine315",*/ "alpine316"]
+}
+
+group "bullseye" {
+    targets = ["bullseye-php-82"]
+}
+
+group "buster" {
+    targets = ["buster-php-82"]
+}
+
+group "alpine315" {
+    targets = ["alpine315-php-82"]
+}
```

Comment:
I think these groups can be removed now?

Reply:
Yes, I'll remove them.